components/plurals/tests/operands.rs

```diff
@@ -31,19 +31,6 @@ fn test_parsing_operands() {
             assert_eq!(operands, test.output.into());
         }
     }
-
-    for test in test_set.floats {
-        let t = test.clone();
-        let operands: PluralOperands = t.output.try_into().expect("Failed to parse to operands.");
-        let expected: f64 = t.input.abs();
-        let actual = (operands.n() - expected).abs();
```

Comment:
Suggestion: manually calculate `n` here, using the same code that we previosly had in `n()`

```
        let fraction = self.t as f64 / 10_f64.powi(self.v as i32);
        self.i as f64 + fraction
```